perceval/backends/github.py

```diff
@@ -39,11 +40,24 @@
 
 GITHUB_URL = "https://github.com/"
 GITHUB_API_URL = "https://api.github.com"
-
+RATE_LIMIT_MIN = 10  # Min rate limit before sleeping until rate limit reset
 
 logger = logging.getLogger(__name__)
 
 
+class RateLimitError(BaseError):
+
+    message = "%(cause)s %(reset_seconds)s seconds for rate reset"
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self._reset_seconds = kwargs['reset_seconds']
+
+    @property
+    def reset_seconds(self):
```

Comment:
Rename this to `seconds_to_reset`. This is a property or an attribute. With the `reset_` prefix it looks like more that calling this will reset something to seconds. You should also rename the attribute in all the occurrences on this class.